space-template.yaml

```diff
@@ -0,0 +1,172 @@
+---
+# These define the potential field types that ca be used in the fields of work
+# item types.
+field_types:
+  - &string string
+  - &translatable_string translatable_string
+  - &list list
+  - &timestamp timestamp
+  - &user user
+
```

Comment:
@aslakknutsen right. I've put them here, so we can refer to them via `*user` for example. Having references makes this whole template a bit more solid I think. @michaelvirgil We should maybe describe better why we have those types in the comment.